extension/content_script.js

```diff
@@ -251,11 +256,21 @@ function parseDocument() {
 }
 
 let readableDomIds = parseDocument();
+doc = new Doc(readableDomIds);
 tracker = new Tracker(readableDomIds);
-display = new Display(readableDomIds, speed);
+window.tracker = tracker; 
```

Comment:
is this global variable used?

Reply:
No but it was in the pull request before, this was kinda weird bc I branched this from the PR that you just reviewed before the comments on it, instead of master bc master was a bit behind at the time ... will go back and fix it tho